src/core/utils/helpers/vitest.ts

```diff
@@ -0,0 +1,4 @@
+export const isVitestEnvironment =
+  typeof process !== "undefined" && process.env && process.env.VITEST;
```

Comment:
@kasperbirch1 We need to do it differently and add some extra functionality to the webpack configuration. But let's talk about it, that's easier :)